styl/components/_list.styl

```diff
@@ -0,0 +1,5 @@
+.list
```

Comment:
@macklinu, I'll directly add the `&--building` modifier to Mozaïk, could be useful for other widgets, see https://github.com/plouc/mozaik/blob/master/src/styl/components/list.styl#L52